packages/ckeditor5-paragraph/src/insertparagraphcommand.ts

```diff
@@ -82,3 +101,12 @@ export default class InsertParagraphCommand extends Command {
 		} );
 	}
 }
+
+// Creates bellow position where should be inserted new paragraph.
+export function createPositionBellow( writer: Writer, position: Position, allowedParent: Element ): Position {
+	const parentPath = position.getParentPath();
+	const length = parentPath.length;
+	const path = [ ...parentPath.slice( 0, length - 1 ), parentPath[ length - 1 ] + 1 ];
+
+	return writer.createPositionFromPath( allowedParent, path );
+}
```

Comment:
There's already an API for it - [`Writer.createPositionAfter()`](https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_model_model-Model.html#function-createPositionAfter). I think we should be safe with `position =` _`writer`_`.createPositionAfter( position.parent as Item );`.

BTW - [https://dictionary.cambridge.org/dictionary/english/bellow](https://dictionary.cambridge.org/dictionary/english/bellow) :smile: